caps/eth.md

```diff
@@ -0,0 +1,223 @@
+# The eth/63 protocol
+
+'eth' is protocol on the [RLPx](../rlpx.md) transport that facilitates exchange of
+Ethereum blockchain information between peers. The current protocol version is **eth/63**.
+See end of document for a list of changes in past protocol versions.
+
+### Protocol Semantics
+
+Upon an active session, a Status message must be sent. Following the reception of the
```

Comment:
I am not familiar with Eth protocol. So "Upon an active session" reads to me like "when a session becomes active", then a Status message must be sent. But the sentence following that says that the session is active only *after* some Status message is sent. This seems contradictory.